connectors/sources/google_cloud_storage.py

```diff
@@ -199,8 +185,54 @@ async def _api_call(
                 )
                 await asyncio.sleep(DEFAULT_WAIT_MULTIPLIER**retry_counter)
 
+    def _validate_configurations(self):
+        """Validates whether user input is empty or not for configuration field.
+
+        Raises:
+            Exception: Configured keys can't be empty.
+        """
+        if self.configuration["service_account_credentials"] == "":
```

Comment:
Should we also check for 'None' here?

Reply:
I checked setting empty field from Kibana and we are getting `""` empty string.
I can't think of a scenario when we might get `None` as a value for `service_account_credentials` for now.

Reply:
TBH I don't think we should rely on Kibana implementation and should be more defensive here.

Reply:
Sure, I'll add the check for the `None` also.